README.md

```diff
@@ -206,11 +206,15 @@ using:
 
 ### Multi-project builds
 
-In most projects with subprojects, the top-level project is not a Java
-project.  You should not apply the plugin to such a non-Java project.  Instead, move
-all Checker Framework configuration (the `checkerFramework` block and any
-`dependencies`) into a `subprojects` block, and do not apply the
-plugin to the top-level project. For example:
+In a project with subprojects, you should apply the project to each Java
+subproject (and to the top-level project, if it is a Java project).  Here are
```

Comment:
My only concern with this change is that this draft de-emphasizes the most common mistake that users make (applying the plugin to the top-level, non-Java project that exists in most multi-project Gradle builds). In the previous text, that was clear: "You should not apply the plugin to such a non-Java project." In this draft, it is instead implied.

I think this can be fixed with one more clause that emphasizes that most users do not want to do what's in the parentheses here. Maybe something like "(and to the top-level project, if it does not follow Gradle convention and is itself also a Java project)"?